bool detect
---
bool bottle_hand
bool bottle_tabletop
bool hand_tabletop
bool succeed
